js/longtable/longtable.js

```diff
@@ -46,7 +46,7 @@ $.fn.longtable = function(fields, options, data) {
       });
       if (id !== undefined) {
         el.addClass("row").addClass("row_"+id);
-        el.css({position: "absolute", top: options.row_h * (id+1)});
+        //el.css({position: "absolute", top: options.row_h * (id+1)});
```

Comment:
nuke these comments, the fact that this was done js is a bug anyway.

Reply:
done